src/OpenTelemetry/Trace/ActivitySourceAdapter.cs

```diff
@@ -186,5 +189,10 @@ private void RunGetRequestedDataOtherSampler(Activity activity)
                 }
             }
         }
+
+        private bool IsParentSpanIdEmpty(Activity activity)
```

Comment:
Might be worth inlining this? Or maybe move the logic up into the `if`. My thinking here is this is hot path and the method doesn't do a whole lot.

Reply:
Thanks, just moved